cypress/support/commands.js

```diff
@@ -27,4 +27,19 @@ Cypress.Commands.add("login", (username, password) => {
     cy.get('#username', { timeout: 5000 }).type(username);
     cy.get('#password').type(password,{log:false});
     cy.get('#submit').click();
+} )
+Cypress.Commands.add("SortFilterSelection", () => {
+    cy.contains("Sort").click({ force: true });
+
+        // Get the open dropdown menu items.
+        cy.get("ul[role='listbox'] li button").then((elements) => {
+            // Get a random number using the max number of elements.
+            const max = elements.length;
+            const min = 0;
+            const randomNumber = Math.floor(Math.random() * (max - min) + min);
+
+            // Wrap the DOM element so we can execute new cypress commands.
+            // Use the randomized number to select a DOM element, then click it.
+            cy.wrap(elements[randomNumber]).click({ force: true });
```

Comment:
@mudit94 i'm not sure if I understand this correctly 

```
Math.floor(Math.random() * (1 - 0) + 0)
```

will always be zero so the max element will be never selected

Reply:
@mudit94 Maybe a screenshot should help me understand this better

Reply:
Also if there aren't any elements would this fail automatically?

Reply:
random() generates either 0,1.

Reply:
This would not fail as it is clicking on either of the options that come as a result of clicking on Sort button